Catalan translation

AinaSG opened the pull request:
Hope it works well :) 

Thanks! 

As a note, in Catalan only the simple `"` are accepted. I used `»` where it was used in the original version to not break the format, but in the final typeset, if possible, `"` should be used. Thanks!

Catalan is a gendered language so where possible I added both endings with a slash: `Jugador/a` , `Jugadors/es` and `Mateix/a`. It's not gramatically correct but since the author specifically requested to use non-gendered language, it's the closest I could get. If it breaks the typeset, removing the `/`and whats after it should be OK and masculine is generally understood as neutral. If it looks good, then I personally think that leaving the `/` in is perfectly fine.

Thanks again for the documents :) Have a great day and many Yakumans!

AinaSG committed:
Translate initiation.txt into Catalan

AinaSG committed:
Translate score.txt into Catalan

AinaSG committed:
Equate capitalization in introduction with the original text

AinaSG committed:
Translate yakus.txt into Catalan

AinaSG committed:
Fix gender in a couple lines

Rusca8 commented:
Omg. Just finished translating it myself thinking no one had done it. :_)

I think I have mostly the same, but after checking this one I feel like I got some of the translations less literal to the english in order to prioritize the catalan way of saying things in other games, the literal japanese meaning or simply a more comprehensible description of the reality of the yaku.

Examples where I differ:
- "Sequences" as "Escales" (since it's the default in other games).
- Being intentional about the meaning of "turn/round" throughout ("Ronda" every time tiles are shuffled, "Volta" for every time everyone has discarded once, "Part" for each of the "East"/"South" halves of the game, as in sports, etc)
- "Round Wind" as "Vent Prevalent" (which is the standard in spanish mahjong and removes ambiguity on the meaning of "round"), and "Player Wind" as "Vent Propi" (which is usually the translation in spanish mahjong and also is inherently gender-neutral)
- Choosing different words for "Drawing" vs "Calling" (I used "Robar" for "Draw", since it's the standard in board-games, and used "Prendre" as "Calling", since it means taking from someone else).
- "Suit" as "Coll", which is the proper catalan translation ("Pal" is wrongly taken from spanish)
- Yaku names where the name describes the yaku (for example, "Three Colored Sequences" as "Escala en Tres Colors", which describes it better than the english version).
- "Heavenly Hand" as "Regal dels Déus" instead of "Mà Celestial" (since the japanese meaning is closer to something like "heavenly blessing")
- Choosing "Nine Lanterns" over "Nine Gates" because both exist in english but the former is the original Japanese.
- "Pinfu" as "Mà Plana (Zero Minipunts)" in order to both use the english/french typical translation and also hint at how it works
- "Menzen Tsumo" as "Totes Teves" (since it's just wining without any tile from the other players)
- Using "Robant" more than "Robatori", which to me sounds kinda weird in board game lingo.

Idk. I wouldn't want to push mine over this one given this has been published here first. 
But maybe I'll upload it just so that we can check both?

zessx commented:
Many thanks to both of you for these translations, I'll take some time to review both proposals and integrate it soon.   
I may have a few questions later (arbitration), @Rusca8 as you're the only one used to GitHub, don't hesitate to answer.

ℹ️  Side note FYI, this project is starting to go beyond what I was expecting, it is currently based on a single Figma file with a few manual tweaks for each language (mostly because spacing), and I need to go through all sheets (almost 20 now) one by one when a change occurs. I think I will have to quickly rework the whole thing to have an HTML-based project, which would highly fasten new integrations and future changes 😅

AinaSG commented:
Hi! Thanks!

@Rusca8 I based my translation on the words I use with my group of players. I guess all groups are different haha.

`Seqüencies vs Escales `- I agree Escales makes sense in games context. :)

`Volta / Ronda...` - Sounds good 👍 

`"Round Wind" as "Vent Prevalent"` Yes, this is better than mine.

`"Player Wind" as "Vent Propi" ` I find this might cause confusion when talking about a player's wind that is not the reader? But i's a minor change that everyone can understand.

`""Drawing" vs "Calling"` True, that needs to be made sure its consistent. Sometimes one gets blind with familiarity.

`"Suit" as "Coll"`, Never heard this word used in cards. I personally would have never guessed what "Coll" meant in this context and I asked around my groups and no one else knew it either. But it's true its the word for it in the dictionary. I'm a bit torn on this because I want the document to be useful and everyone would know what "pal" is but not what "coll" is (Or maybe my Central Dialact is showing and its a super common word everywhere else). But at the same time I agree that language correctness is valuable...

`Yaku names` - Yes, those can be reviewed.

`"Mà Celestial"` - This is a tranlsation by my chinese co-players, they would interpret it as "Ma celestial" + "Ma terrenal" + "Ma del home" (As "You won by the action of a heavenly hand") and my group just thinks it sounds really epic like this haha, but I'm 100% for standarization.

`"Nine Lanterns" over "Nine Gates" ` - Nine Gates is the way to go IMO, it really explains the Yaku beter, there is literally 9 ways you can win the hand. It's also the most common name in all english translations and in the Federation rules.

`"Pinfu" as "Mà Plana (Zero Minipunts)"` - This one is a tricky one, because everyone calls it just Pinfu, untranslated, in the federation rules is like this even. I would consider this a "Japanese Named Yaku" (Riichi, Ipatsu, Pinfu are the 3 yakus that the fedaration uses the japanese name for and I agree on following that, specially if this document wants to introduce players that eventually want to play online or by federation games).

`"Menzen Tsumo" as "Totes Teves"` - This sounds goofy to me... Federation rules call it "Fully Concealed hand" and the original document calls it "All Concealed Self Draw" , this is why I translated it to "Mà Tancada amb Robatori Propi". I feel its not too long, similar to other translations, and very descriptive. 

Its nice to compare translations, maybe you can push it so we can diff it and get to the perfect one? :D **Edit:** Ah, I see it you have there. Will try to see if we find an inbetween if I have the time. 🤔 

------
@zessx I am also familiar with github use, just I use a private instance for my day job, so this is why I have little activity here haha. Don't hesitate to ping me if there is any question.

I'm not a graphic designer, but I recently formated and generated some documents for another game using jinja2 templating engine and then rendered them from the generated HTML, it worked really well, co maybe you could look into something like that. As I said I'm not an expert but I was super surprized on how easy it was.

Rusca8 commented:
```"Player Wind" as "Vent Propi"``` In what kind of contexts you imagine it could be confusing? I feel like it'd be understood quite literally as "the player's own wind", which doesn't seem confusing to me, but I may be missing something. 
Also, it's the way the spanish wikipedia translates it (together with "Viento del Asiento", which I guess was translated straight literally from the english "Seat Wind", but to me it feels quite off in spanish).
Also I guess it's not an unsusual wording since we literally use that for football goals (as in, "Gol en Pròpia").

```Coll``` I think is kinda more used towards Girona and such, but yeah I learned the word not that long ago. I guess most people wrongly translate it from the spanish and don't realise we have our own word for it. I think it wouldn't at all be confusing in the cheat sheets themselves, though, since you have the tiles drawn (and also, each of the suits' names) as context for it. People may see the title and be thrown off at first, but won't mess up their understanding (and it'd be a subtle way to teach them the proper word for it).

```Mà Celestial``` Them being chinese makes me doubt, but I wonder where did they take that meaning from. The characters (at least on the Japanese name for the Yaku) have no literal "hand" meaning (the japanese dictionary [jisho](https://jisho.org/search/%E5%A4%A9%E5%92%8C) translates the word as "Blessing from heaven"). The only "Hand" I've found on that comes from the english translation "Heavenly hand", which I bet uses "hand" as in "bunch of tiles you have as a player", not as in "a part of a body" (and thus, makes it actually mean "blessing from heaven", as in "the heavens got you a bunch of nice tiles"). I chose ```Regal``` (Gift) over the literal ```Benedicció``` (Blessing) to keep it shorter.

```Nine Gates``` Oh I didn't realise it's called that way because you have a 9-sided wait. Then it indeed makes more sense to choose the "gates" translation over the "lanterns" one in order to better convey that "9 ways to reach somewhere" meaning.

```Pinfu``` Yeah I thought about that too. If we had to choose one or the other I'd also keep "Pinfu". My line of thought was that we already have that name on the Japanese side of the Cheat Sheet, and we might as well use the translated side to better explain the yaku, specially thinking about the beginners that will use the sheet. I saw that the [riichi wiki terminology table](https://riichi.wiki/List_of_terminology_translations) uses "Flat Hand" as a translated version of "Pinfu" for both english and french, so I used that (and also kept the "Zero Minipunts" as a short summary of the mechanic).

```Menzen Tsumo``` I guess the strict translation from the federation would be ```Mà Totalment Tancada```, which is longer but also a clear description (since "Ron" would in fact open the last meld even if the hand stays closed). The "All Concealed Self Draw" one felt a bit cumbersome to me. I went for "Totes Teves" (All Your Own) because it felt similar to the way tanyao is translated as "Tot Simples" (All Simples), and because it felt like a clear and exact description of what needs to happen (i.e. that you don't use a single tile from the other players). Like, it indeed sounds less serious than "Fully Concealed Hand", but I don't think it sounds more goofy than "All Simples".

AinaSG committed:
Improved with @Rusca8 version

AinaSG commented:
Fantastic! Thank you for your notes.

I have been playing Riichi for over 10 years with a similar group of people, so I guess it's normal some inside language has developend in the group that feels weird when being changed or challenged. So sorry if it feels like I'm a bit defensive on some topics.

--------

`"Vent Propi"` - Ok, "Gol en propia" convinced me... What do you think maybe calling it "Vent de la cadira" as in spanish? I want somehow to make it apparent that is dependent on the position you are seated at. **Edit:** After finishing the edit, Vent Propi sounds good and I'll keep that.

`Mà Celestial` - Maybe just a internal name in the group. I see in the fedaration rules the translation is more similar to yours, so I'll adjust it.

`Coll ` - Ok, yeah, makes sense, I think it's good to use "the correct" word to make it more familiar to players.

`Pinfu` - I still think that since it's universally called "Pinfu" we should keep it like that. I added "Zero minipunts" as part of the explanation of yhe yaku, since it seems important for your group to mention and at least is interensting.

--------

I'm patching my branch with your notes using the diff from your translation. Some things I am personally on the fence, with no concrete order, are:

`Gender Neutrality` - I underatand your point that masculine is the generic... At the same time the Original author requested to use neutral language so thats why I use Jugador/a... I know its a bit halfway, but I guessed this sounded fluent enough without getting too long and I would like to explicitly make evident that this game dos not assume a he/him reader... At least in our group we are a diverse mix and thought it would be nice to make it explicit. Can be easilly changed.

`"Condició de victoria" or "Combinació valida"` Tricky one. I used "Combinació valida" because it makes really obvious that is a certain shape of the hand that we are looking into, and that you can have multiple combinations in a hand, but its true that the shape of the hand is not entirely the requisite to win, also the original text uses "Combination", but I like your translation better I think. I'll change it to yours.

`"El mur del rei"` never heard this translation. Mur Mort / Dead walll is the name that is used in all translations and its descriptive, is a part of the wall the pieces of what are "dead", not going to be taken in regular gameplay. All IRL circles I have been to use "Mur mort".

`"Value triplet"` should be "Trio de valor" not just "Trio", Not all Triplets are equal

`"Escala Sencera" vs "Escala de Color"` -  I used "Escala de Color" because this is the poker name and people are familiar with poker, but yours is more consistent with the rest of the text.

The meaning of `Ronda`: I understand your point of whats a Ronda. In all languages tho, a Round is a complete spin of the winds, so using it differently in Catalan might cause confusion. I still think that being consistent inside the text is more important tho, so I reviewed the text to make it consistent and we can discuss it further. At least in all Tables I played we call: "Ronda" a complete game where you play as each win, and "Ma" every time you mix the tiles. and "Joc" a complete East and South game. It seems to be also the terminology used by the Federation so I'd love to be able to keep it consistent. I changed "Vent de la Ronda" to "Vent Prevalent de la Ronda" to use the word Prevalent witch is part of the international vocabulary aswell. Thanks!

-------

My final goal with the document is to initiate players that can join the reglated competitions in a future, so consistency with Federation Rules is very important...

I'll add the changes as a Commit to this PR, if you want to check them out :)

This is the European Federation rules that are followed in competitions, I guess you have seen them already, but sharing just in case: http://mahjong-europe.org/portal/images/docs/Riichi-rules-2025-EN.pdf

**Edit: Change done, here is the diff:** https://github.com/zessx/riichi/pull/3/commits/2253a6f35647e8b3949522755e0d66d8b6c05b29

Rusca8 commented:
@AinaSG don't worry at all about sounding defensive! I haven't competed at all in Mahjong and just started playing recently. I had the european rules pdf open in a forgotten tab, pending a read.

I come here quite from the side. I've worked as a modern board game expert in a shop that did activities for schools, and so I think I have a kinda good instinct for when the explanations & rules will be easy to understand to newbies, but I don't really know much about the reality of mahjong outside of my bunch of friends that taught me (that is, besides what I discovered after spending a bunch of time researching vocab online while trying to translate).
I also knew a bit of japanese, which helped at navigating old japanese names.

Though, no hard opinions on anything, since on the one hand I'm sure you'll know more about the lingo in official settings, and on the other hand if I really wanted to have something my way when teaching my friends, I could always take the end result and make a slightly altered version for myself.

```Gender Neutrality``` I think that's good too. I just made the choice for practical reasons when I was working on it, but I agree it's also nice to explicitly include diversity. I actually tend to use general feminine when I can, but when space is a concern I sometimes end up folding. ahahah 

```Condició de Victòria``` I also found this kinda tricky. I went this route because I felt like newer players often get confused by having Yakus that aren't a hand shape but instead some other condition. This way it seems like we make sure the emphasis is in "accomplishing one of the missions" more than in "having the proper tiles in your hand".

I got to a similar reasoning on the (quite deliberate) ```Aprofitant que algú altre la descarta``` for the initiation description of ```Ron```. As in, trying to not imply that you "take" that piece (since protocol usually keeps it on the other person's discard, and also, more than one player could Ron it at the same time).

```Mur del rei``` Yeah I agree that may be a bit weird. The Japanese literal is "King's Tiles", so I ended up going for an in-between. But it's true that everyone seems to call that "Dead Wall" in non-japanese languages. Maybe I went a bit too far in going for the original japanese meanings.

```Value Triplet``` I think I do have ```Trio de Valor```! But yes I agree.

```Escala Sencera``` I thought about that one. What made me decide for this is that every sequence in Mahjong is already from the same suit (so it felt more useful to put the emphasys on the "has to have all the numbers" part than the "has to be the same suit" part).

```Ronda``` Was an absolutely tricky one, since a lot of things could be called ```Ronda```. I could have called "Ronda" a full spin of the winds, or a full hand, or each player playing once, so I ended up making a (maybe too arbitrary) choice just so that I could have different words for each of those. I like your choice of words too, though. Having ```Mà``` for every time you shuffle seems a good way to remove the ambiguity of "Round". And I guess we'd keep "Volta" for when every player discards once? (cause the english uses "turn" but in catalan "Torn" is always used to mean that only one people plays once)
Also, ```Vent Prevalent de la Ronda``` sounds good to me if it fits the available space in the cheat sheet. Like, I guess the whole concept is ```Vent Prevalent de la Ronda```, and even when you'd shorten it to ```Vent Prevalent``` it'd be implied that it's ```de la Ronda```, so we might as well be explicit about it.

Rusca8 commented:
@zessx as for the parametrization of the design, I'm familiar with Reportlab (which works nice but admitedly is a bit clumsy) and also with that same jinja2/html.

If you try to convert things and need an extra hand I might be able to help. 🫡

Rusca8 commented:
@AinaSG I'd maybe try to use ```Robant de...``` instead of ```Robatori``` in the places it appears? For some reason ```Robatori``` feels off to me when talking about board games.

For Menzen, if you have differentiated the words for ```Draw``` and ```Call``` then you'd be free to simply say "Mà Tancada Robant l'Última```.

AinaSG commented:
@Rusca8 Yes, you are right, all the "Robatori" also sounded weird to me, but could not think of something better. Maybe Robant does the job. If I have some free time tomorrow Ill review it. :).

Rusca8 commented:
It seems there's only like three different places where it still remains (though some of them appear more than once).

It could be something like this:
"Mà Tancada amb Robatori Propi" > "Mà Tancada, Robant l'Última"
"...en furiten fins al teu següent robatori" > "...en furiten fins robar o prendre fitxes"
"Robatori del Mur Mort" > "Robant del Mur Mort"

Also you have an instance something like "robatoi" somewhere there. In case you find/replace 🫡

PD: I think it'd be also good to always use the same word for "Call", be it "agafar" or "prendre". Now there's both depending on the section and that may dilute the meaning of things for new players.

Rusca8 commented:
Some details I've seen after checking the version a bit more thoroughly:

**initiation:**
140 ```per com``` > ```com``` (typo from changing your mind on phrasing I guess)
170 ```torn ininterromput``` > ```volta ininterrompuda```
184 I'd maybe say something like ```Si has descartat en algun moment una fitxa que podria completar...```, cause I think the current wording (```has descartat ... que podria haver completat```) is ambiguous on whether the older discards should be taken into account (i.e. "podria haver completat" might as well mean that it could have completed the hand at that point in time, and "has descartat" without "en algun moment" may make it seem like it starts to count once in tenpai?) idk, I'd be more specific just in case.
192: I guess it's "mà" instead of "partida"

**yakus**
72: This should be ```prenent d'un quartet``` (or ```agafant d'un quartet```, depending on the choice above) if we want to be coherent with ```Robar``` being only used for ```Draw``` and not ```Call```.
74: Following 72, ```Només es pot prendre d'un quartet```, for the same reason
82&86: ```volta``` instead of ```torn``` (though careful, since there's a properly used ```torn``` at the end of the sentence.
164: A sneaky ```Pal``` > ```Coll``` escaped revision

AinaSG commented:
👌 

So Translation guidelines:

**Divisions of gameplay**
A single take and put tile down: `Torn`
This done by all players: `Volta`
Each time tiles are shuffled: `Mà`
The entire set of hands that have the same prevalent wind: `Ronda`
An East and South game: `Joc`

**Mahjong concepts**
The tiles: `Fitxes`
Suits: `Colls`
Pairs: `Parelles`
Triplets: `Trios`
Quads: `Quartets`
Sequences: `Escales`
Take from the wall / Tsumo win: `Robar`
Take from another player's discards/ Ron win: `Prendre`
Seat wind: `Vent Propi`
Round Wind: `Vent Prevalent de la Ronda`

New version in the comming commit.

AinaSG committed:
Standarization of terms + some errors fixed

zessx commented:
Wow, you're doing a great job here! 

@Rusca8 Should I close your PR, as you both seem to use this one to agree on a standardized version?

zessx commented:
Also, should I credit you (both) under your GitHub pseudonyme, or do you want to use another name?

AinaSG commented:
Sorry for the turbospam we had here haha.
For me Aina SG or just Aina is fine for a credit. Thanks :)

zessx commented on i18n/translations/ca/score_ca.txt:
```diff
@@ -0,0 +1,182 @@
+transl "Pinfu Obert"
+origin "Tsumo (/wo pinfu)"
+transl "Tsumo (sense pinfu)"
+origin "Value pair"
+transl "Parella de valor"
```
I'm missing a new translation since EMA RCR 2025, as we no longer counts +4 for a double Wind pair (dabuton/dabunan), but only +2:

> Not cumulative

I went for "No acumulatiu", is that correct?

Rusca8 replied:
Yes! ```No acumulatiu``` is right

AinaSG replied:
Ah, did not know about this change. Makes sense, but kinda sad because its very rare and funny when it happens.
I don't know if `No acumulatiu `or `No acumulable` is better... maybe in context, but the difference is extremly minor.

Rusca8 replied:
Checking the dictionary, it feels to me like ```No acumulable``` is indeed the better one.
(I guess it's a slight difference, but "acumulable" appears there with this exact meaning, and "acumulatiu" is instead fuzzier.)

zessx commented on i18n/translations/ca/score_ca.txt:
```diff
@@ -0,0 +1,182 @@
+origin "Ippatsu"
+transl "Ippatsu"
+origin "All Concealed Self Draw"
+transl "Mà Tancada i Robant l'espera del mur"
```
Wow, you're really using this terminology when stating your Yaku?

Rusca8 replied:
That's a literal for "Concealed Hand and Drawing the Wait From the Wall".

It feels kinda long to me too. Up there I was going for "Mà Tancada, Robant l'Última" ("Concealed Hand, Drawing the Last One", as in, not Calling it). Though I'd maybe do just "Mà Totalment Tancada" (Completely Concealed Hand). Which I think does already quite explain it (Since "Ron" does open the tiles it melds)

AinaSG replied:
> Wow, you're really using this terminology when stating your Yaku?

Honestly, in my group we call this `Menzen Tsumo` - Since the english version used a translated descriptive name, I did the same

AinaSG replied:
`"Mà Totalment Tancada"` Sounds right.

zessx commented on i18n/translations/ca/initiation_ca.txt:
```diff
@@ -0,0 +1,224 @@
+transl "Vermell"
+origin "Round Wind"
+transl "Vent Prevalent de la Ronda"
+origin "1st = East"
+transl "1a part = Ronda Est"
+origin "2nd = South"
+transl "2a part = Ronda Sud"
```
I'll remove "Ronda", as we don't have much place there, and the title is already stating "Vent Prevalent de la Ronda"

Rusca8 commented:
For my credit you can write just "Rusca".

Thanks!

zessx commented:
The 3 sheets have been published on https://zes.sx/riichi/, thank you very much for you work 🙏

AinaSG commented:
Looks Great! Thank you very much!

Rusca8 commented:
Hi! Thanks a lot!

I found a couple of typos after checking the sheets (in case you want to fix things now that you are at it).

**Yakus**
- We forgot to capitalize the ```V``` in the ```Trio de Valor``` of this sheet (the other sheets have it properly capitalized)

**Initiation**
- You left a (german?) ```Wertlose Hand``` on the ```Pinfu``` title.

**Score**
- There's a weird space before the colon on the ```Honba``` section (```comptador :``` instead of ```comptador:```).

Apart from this, there's a couple of things we didn't see that I'd maybe tweak if @AinaSG agrees.

On the **Yakus** sheet:
- ```Un Quartet tancat només es pot prendre``` sounds to me like we're talking about taking the whole quad. I'd maybe say ```Només es pot prendre d'un quartet tancat``` instead (while keeping the second half of the sentence as it is), so that it's clearer that we only take one tile.
- Maybe it shoud be ```una Terminal``` instead of ```un Terminal```? (since I guess the unspoken word here is ```fitxa```, which would be feminine)

On the other hand, I still think the wording for the first furiten condition (**initiation** sheet) is ambiguous unless you already know the rules (i.e. the way it's written, it sounds to me like you get into furiten when you choose not to take an early win, and also that you'd stop being in furiten on any change of waiting tiles since the discard took place). It's kinda small once you know how to play but I think the nuance matters when teaching newbies.

I'd change it to something more on the lines of:
> Si has descartat en algun moment una fitxa **que ara et serviria per completar la mà** ...

PS: not sure if it's intentional or not, but just so you're aware of it, the flag you chose for the catalan language on the website is actually the flag of the catalan regional struggle for independence (the official neutral catalan flag is the one with only the four red lines over a yellow background). Most of the catalans won't care at all (or, quite often, we'll actually find it amusing that you accidentally took a stance in our favor ahahah), but just, be aware that some spanish people (and some against-independence catalans) may be pissed off by it being the non-neutral choice.

AinaSG commented:
> Un Quartet tancat només es pot prendre sounds to me like we're talking about taking the whole quad. I'd maybe say Només es pot prendre d'un quartet tancat instead (while keeping the second half of the sentence as it is), so that it's clearer that we only take one tile.

Agree.

> Maybe it shoud be una Terminal instead of un Terminal? (since I guess the unspoken word here is fitxa, which would be feminine)

Don't agree. 
Here Terminal is not an adjective (witch would take the gender of the missing word) but a Name. 

`El Terminal`, as in the ends of a line/series/network connection/cable. 

In Spanish contexts is always translated in masculine, even when in spanish both versions are correct as names, witch puts more enphasis that it's used as a name not as an adjective, and as a name, only the Masculine version is accepted in Catalan. 

About the flag. I found it funny, and I'm not too politically aligned in this topic, but it left me wondering if it was on purpose or you did not know about the use. haha

zessx commented:
Thanks for the quality check:
- ✅ Capitalize the V in the Trio de Valor (Yakus)
- ✅ Remove german leftover (Initiation)
- ✅ Remove extra space (Score)
- ✅ Rephrase Chankan description (Yakus)
- ❌ Rephrase "una Terminal"

About the flag, I was taught the difference when living in Barcelona for a few months (something like 15 years ago), and I don't mind standing for this cause at all, but tbf the reason is simpler: only this version was available 😬
Anyway, this make me realize flag icons licence was not checked, so I may have to change them.
